bin/vue-init

```diff
@@ -12,7 +12,7 @@ var inquirer = require('inquirer')
 var request = require('request')
 var logger = require('../lib/logger')
 var generate = require('../lib/generate')
-var checkVersion = require('../lib/check-version')
+var { checkVersion, warnForVue2Version } = require('../lib/check-version')
```

Comment:
I love destructuring too, 😃  but it's unfortunately [only available in Node 6+](http://node.green/#destructuring--declarations) and I believe we're supporting down to v4.